interface/forms/newpatient/common.php

```diff
@@ -291,39 +289,38 @@ function cancelClickedOld() {
     }
     ?>
 </head>
-<body class="body_top" <?php echo $body_javascript; ?>>
-    <div id="container_div" class="<?php echo attr($oemr_ui->oeContainer()); ?>">
+<body <?php echo $body_javascript; ?>>
+    <div id="container_div" class="<?php echo attr($oemr_ui->oeContainer()); ?> mt-3">
         <div class="row">
             <div class="col-sm-12">
                 <!-- Required for the popup date selectors -->
                 <div id="overDiv" style="position: absolute; visibility: hidden; z-index: 1000;"></div>
-                <div class="clearfix">
-                    <?php echo $oemr_ui->pageHeading() . "\r\n"; ?>
-                </div>
+                <?php echo $oemr_ui->pageHeading() . "\r\n"; ?>
             </div>
         </div>
         <form class="mt-3" id="new-encounter-form" method='post' action="<?php echo $rootdir ?>/forms/newpatient/save.php" name='new_encounter'>
             <?php if ($viewmode && $mode !== "followup") { ?>
-                <input type=hidden name='mode' value='update' />
-                <input type=hidden name='id' value='<?php echo (isset($_GET["id"])) ? attr($_GET["id"]) : '' ?>' />
+                <input type='hidden' name='mode' value='update' />
+                <input type='hidden' name='id' value='<?php echo (isset($_GET["id"])) ? attr($_GET["id"]) : '' ?>' />
             <?php } else { ?>
                 <input type='hidden' name='mode' value='new' />
             <?php } ?>
             <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
 
             <?php if ($mode === "followup") { ?>
-                <input type=hidden name='parent_enc_id' value='<?php echo attr($encounterId); ?>' />
+                <input type='hidden' name='parent_enc_id' value='<?php echo attr($encounterId); ?>' />
             <?php } ?>
 
             <fieldset>
                 <legend><?php echo xlt('Visit Details') ?>
                     <small>
-                        <?php echo $encounter_followup ? (xlt("Follow up for") . ": " . $encounter_followup . " Dated: " . $followup_date) : ''; ?></small>
+                        <?php echo $encounter_followup ? (xlt("Follow up for") . ": " . $encounter_followup . " Dated: " . $followup_date) : ''; ?>
+                    </small>
                 </legend>
                 <div id="visit-details" class="px-5">
                     <div class="form-row align-items-center">
                         <div class="col-sm-2">
-                            <label for="pc_catid" class="oe-text-to-right"><?php echo xlt('Visit Category:'); ?></label>
+                            <label for="pc_catid" class="text-right"><?php echo xlt('Visit Category:'); ?></label>
                         </div>
```

Comment:
Hi @bradymiller and @tywrenn 

Although I convert the `oe-text-to-right` class to the bootstrap `text-right` class, but I found the OpenEMR-interface document write detail about the `oe` class, Does the document outdated?

https://github.com/openemr/openemr/tree/master/interface#special-classes

Reply:
@stu01509 It _does do_ what the documentation says, _however_, this was used before BS4 was added. Now we can replace it with `text-left` and `text-right`.